packages/vaadin-text-field/theme/material/vaadin-text-area-styles.js

```diff
@@ -9,16 +9,11 @@ registerStyles(
       box-sizing: border-box;
     }
 
-    .textarea-wrapper {
+    [part='input-field'] [part='value'] {
```

Comment:
Please also add `[part='input-field'] ::slotted(textarea)` (even though in Vaadin 21 it's not a public feature).

Reply:
Done